### PR TITLE
stb_vorbis: change namespace from stb_ to libxmp_ for public functions.

### PR DESCRIPTION
**Pull request by sezero**

### CODE REVIEW

**Review by AliceLR — APPROVED** [review]
This seems like a reasonably non-intrusive way to handle this issue. Should probably wait until after #512 is merged so the seek defines can be dropped.

**Comment by sezero** [comment]
> This seems like a reasonably non-intrusive way to handle this issue.

Agreed

> Should probably wait until after #512 is merged so the seek defines can be dropped.

No need: if in the unlikely case we re-enable anything, they will still be mangled properly.

**Pull request merged**
